### PR TITLE
Catalog of my things

### PR DESCRIPTION
**Pull request by TamaraDimov**
In this PR We built the whole CLI application, by including the following features:

-Ability to add new books, music albums, and games to the program.
-Ability to list all books, music albums, and games stored in the app.
-All data is preserved locally by using plain text files.
-We also added an analogous SQL schema to preserve data in the server in the future.
🎥 Watch a presentation video:
[Click here](https://drive.google.com/file/d/1hGp6IeNX1352gGhdbjxTs9hYwF0qaq8v/view?usp=sharing)

ℹ️ To use this app:

Clone the repo by running https://github.com/TamaraDimov/Catalog-of-my-things.git
Move to the new folder by running cd catalog-of-my-things
Execute the program by running ruby main.rb

### CODE REVIEW

File: item.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def to_h
+    {
+      id: @id,
+      publish_date: @publish_date.to_s,
+      archived: @archived
+    }
+  end
+
+  attr_writer :label
+end
```
Comment (VICTOR-CHUKWU) [inline item.rb:36]: - [ ] hello team great work on your project and testing it properly, but I think you skipped testing this file, kindly add a test for this file :+1:

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require './Start'
```
Comment (VICTOR-CHUKWU) [inline main.rb]: - [ ] You have a little typo error here, the file name you are importing is `start` not `Start` just the capital letter `S` :+1:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @TamaraDimov  and @kifle23  and @Nasirkhan294 , 

## SATAUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by kifle23** [comment]
Hello @VICTOR-CHUKWU thank you for your review. We have updated the code as per your change request 🙏

File: item.rb (hunk 1 above)
Comment (Nasirkhan294) [reply item.rb:36]: Hi @VICTOR-CHUKWU, Thanks for highlighting the issues. We will fix them and submit the pull request for another evaluation.

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @TEAM    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  . 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] Professional Readme file 
- [x] No linter errors
- [x] Followed project requirements 💯 



 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Comment by kifle23** [comment]
On behalf of the team thank you @KayLemba for your approval 🙏

**Pull request merged**